src/resources/boards/board.controller.js

```diff
@@ -0,0 +1,62 @@
+const BoardService = require('./board.service');
+const { successfulOperation, successfulCreated } = require('../../messages/success');
+const { invalidToken, notFound } = require('../../messages/errors');
+
+
+class BoardController {
+
+    static async getAllBoards(req, res) {
+        try {
+         const boards = await BoardService.getAll();
+         successfulOperation(res)
+            return res.end(JSON.stringify(boards))
+        } catch(e){
```

Comment:
Better to pass catched e to invalidToken function and there write something like:
`JSON.stringify({ message: e.message })`
it'll be more dynamic